arch/HPPA/HPPADisassembler.c

```diff
@@ -1671,6 +1681,7 @@ static bool decodeShexdep0(cs_struct *ud, MCInst *MI, uint32_t insn) {
             CREATE_GR_REG(MI, r2);
             if (ext <= 0x01) {
                 CREATE_CR_REG(MI, 11);
+                (&MI->hppa_ext)->cmplt = true;
```

Comment:
Please move this dereference pattern (`&MI->hpppa_ext`) into an inline function or macro.